PROCESSORS.md

```diff
@@ -1292,11 +1293,13 @@ Listens for incoming TCP connections and reads data from each connection using a
-|-------------------------------|---------------|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Listening Port**            |               |                  | The port to listen on for communication.                                                                                                                                                      |
-| **Max Batch Size**            | 500           |                  | The maximum number of messages to process at a time.                                                                                                                                          |
-| **Max Size of Message Queue** | 10000         |                  | Maximum number of messages allowed to be buffered before processing them when the processor is triggered. If the buffer is full, the message is ignored. If set to zero the buffer is unlimited. |
+| Name                          | Default Value | Allowable Values           | Description                                                                                                                                                                                      |
+|-------------------------------|---------------|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Listening Port**            |               |                            | The port to listen on for communication.                                                                                                                                                         |
+| **Max Batch Size**            | 500           |                            | The maximum number of messages to process at a time.                                                                                                                                             |
+| **Max Size of Message Queue** | 10000         |                            | Maximum number of messages allowed to be buffered before processing them when the processor is triggered. If the buffer is full, the message is ignored. If set to zero the buffer is unlimited. |
+| SSL Context Service           |               |                            | The Controller Service to use in order to obtain an SSL Context. If this property is set, messages will be received over a secure connection.                                                    |
+| Client Auth                   | NONE          | NONE<br/>WANT<br/>REQUIRED | The client authentication policy to use for the SSL Context. Only used if an SSL Context Service is provided.                                                                                    |
 
```

Comment:
I know this is not part of this PR, but I think we should list the output attributes (tcp.port, tcp.sender) here aswell similar to ListenSyslog. So the feature is apparent to the user.

Reply:
Updated in d0e2ac6c3abeff8d46f8e15fd80816dd86fa9b13